getting-started.adoc

```diff
@@ -1,3 +1,251 @@
+
+Now that you've taken an offer, it's almost time to pay the seller.
+
+First, before showing you the seller's payment details, Bisq will wait for the Bitcoin network to confirm the trade's transactions. On average, this takes 10 minutes.
+
+Then, you'll see a pop-up with the seller's details. Go ahead and pay the seller through the agreed payment method.
+
+[NOTE]
+.Confirmations?
+====
+Each confirmation makes a bitcoin transaction exponentially harder to reverse. You can wait for as many confirmations as you like before sending payment—just note the remaining time indicator. The seller must _receive_ your payment before that timer runs out.
+
+.Make sure the seller receives your payment before this timer runs out.
+image::remaining-time-to-pay.png[Send payment quickly]
+
+Learn more about confirmations https://en.bitcoin.it/wiki/Confirmation[here^].
+====
+
+--
+
+=== 5. Mark payment as sent
+
+[.float-group]
+--
+
+[.right.text-center]
+.Mark payment as sent.
+image::mark-payment-sent.png[Mark payment as sent,400,400]
+
+Once you've sent your payment, be sure to mark the payment as sent in Bisq by clicking the `Payment started` button.
+
+It's easy to forget this step, but remember: Bisq doesn't integrate with national currency payment methods in any way—so Bisq won't know you've sent your payment until you say so.
+
+When you click `Payment started`, the seller will receive a notification that lets them know the payment is on the way. They'll be on the lookout for it now.
+
+--
+
+=== 6. Complete trade
+
+[.float-group]
+--
+
+[.right.text-center]
+.Trade complete.
+image::complete-trade.png[Complete trade,400,400]
+
+When the seller receives your payment, they'll mark it as received in Bisq. Bisq will then complete the trade by releasing the bitcoin being traded to you along with your security deposit.
+
+You can keep your new bitcoin in your built-in Bisq wallet by clicking `Move Funds to Bisq Wallet` or send it elsewhere by clicking `Withdraw to External Wallet`.
+
+--
+
+Congratulations. You've just completed your first trade on Bisq!
+
+== Next steps
+
+It's *crucial* that you secure your Bisq wallet. Without a password, your wallet data is stored on your hard drive unencrypted. That's _dangerous_.
+
+Please follow our https://docs.bisq.network/secure-wallet.html[short guide here] to set a password and encrypt your wallets.
+
+== Get help
+
+If you get stuck, reach out! There's a community of people to help you on the https://bisq.community/[Bisq forum^], the https://www.reddit.com/r/bisq/[/r/bisq subreddit^], and the https://bisq.network/slack-invite[Bisq Slack team^].
+
+== Stay in touch
+
+You can get updates on https://twitter.com/bisq_network[Twitter^], https://www.facebook.com/bisqnetwork/[Facebook^], and https://www.youtube.com/c/bisq-network[YouTube^].
+
+If you really like Bisq, https://docs.bisq.network/contributor-checklist.html[consider contributing]! Even if you're not a developer, there's lots you could do.
```

Comment:
Figured it worth adding a short bit about contributing (why not). I _really_ want to add something about the DAO here to further spark peoples' interest, but haven't figure out a way to convey it that isn't overkill.